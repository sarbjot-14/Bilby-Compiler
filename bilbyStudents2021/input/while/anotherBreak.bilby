main{
	mut count := 0;
	mut count2 := 0;
	while (count < 5) {
		print "outer"$s count $n;
		count := count +1;
		while (count2 < 7) {
			print "inner" $s count2 $n;
			count2 := count2 +1;
			
		}
		count2 := 0;
		
		if(count ==3){
				break;
			}
		
		
	}
}