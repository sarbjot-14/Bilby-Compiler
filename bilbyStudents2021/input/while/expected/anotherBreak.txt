outer 0
inner 0
inner 1
inner 2
inner 3
inner 4
inner 5
inner 6
outer 1
inner 0
inner 1
inner 2
inner 3
inner 4
inner 5
inner 6
outer 2
inner 0
inner 1
inner 2
inner 3
inner 4
inner 5
inner 6
